bundles/org.openhab.binding.yamahamusiccast/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,236 @@
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="playerControls">
+		<label>player Controls</label>
+		<channels>
+			<channel id="channelPlayer" typeId="channelPlayer"/>
+			<channel id="channelArtist" typeId="channelArtist"/>
+			<channel id="channelTrack" typeId="channelTrack"/>
+			<channel id="channelAlbum" typeId="channelAlbum"/>
+			<channel id="channelAlbumArt" typeId="channelAlbumArt"/>
+			<channel id="channelRepeat" typeId="channelRepeat"/>
+			<channel id="channelShuffle" typeId="channelShuffle"/>
+		</channels>
+	</channel-group-type>
+
+
+
+
+	<channel-type id="channelPower">
+		<item-type>Switch</item-type>
+		<label>Power</label>
+		<description>Power channel</description>
+	</channel-type>
+	<channel-type id="channelMute">
+		<item-type>Switch</item-type>
+		<label>Mute</label>
+		<description>Mute channel</description>
+	</channel-type>
+	<channel-type id="channelVolume">
+		<item-type>Dimmer</item-type>
+		<label>Volume</label>
+		<description>Volume channel - percentage</description>
+		<state min="0" max="100"/>
+	</channel-type>
+	<channel-type id="channelVolumeAbs">
+		<item-type>Number</item-type>
+		<label>Volume</label>
+		<description>Volume channel - Absolute value</description>
+	</channel-type>
+	<channel-type id="channelInput">
+		<item-type>String</item-type>
+		<label>Input</label>
+		<description>Input channel</description>
+	</channel-type>
+	<channel-type id="channelSoundProgram">
+		<item-type>String</item-type>
+		<label>SoundProgram</label>
+		<description>SoundProgram channel</description>
+	</channel-type>
+	<channel-type id="channelSelectPreset">
+		<item-type>String</item-type>
+		<label>Select Net Radio/USB Preset</label>
+		<description>Select Net Radio/USB Preset channel</description>
+	</channel-type>
+	<channel-type id="channelPlayer">
+		<item-type>Player</item-type>
+		<label>Player for Net Radio/USB</label>
+		<description>Player for Net Radio/USB channel</description>
+	</channel-type>
+	<channel-type id="channelSleep">
+		<item-type>Number</item-type>
+		<label>Sleep</label>
+		<description>Sleep Time in minutes</description>
+		<state>
+			<options>
+				<option value="0">No Sleep</option>
+				<option value="30">30 min</option>
+				<option value="60">60 min</option>
+				<option value="90">90 min</option>
+				<option value="120">120 min</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="channelMCServer">
+		<item-type>String</item-type>
+		<label>Server for MusicCast Link</label>
+		<description>Select IP of device to be Server for MusicCast Link - v2</description>
+	</channel-type>
+	<channel-type id="channelUnlinkMCServer">
+		<item-type>Switch</item-type>
+		<label>Deactivate MusicCast Link on Server</label>
+		<description>Deactivate MusicCast Link on Server</description>
+	</channel-type>
+
+	<channel-type id="channelRecallScene">
+		<item-type>Number</item-type>
+		<label>Scene selection</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Done